src/interactivity/win32/UiaTextRange.cpp

```diff
@@ -145,10 +144,10 @@ std::deque<UiaTextRange*> UiaTextRange::GetSelectionRanges(_In_ IRawElementProvi
     return ranges;
 }
 
-
 UiaTextRange* UiaTextRange::Create(_In_ IRawElementProviderSimple* const pProvider)
 {
-    UiaTextRange* range = nullptr;;
+    UiaTextRange* range = nullptr;
+    ;

```

Comment:
hey this one seems wrong

Reply:
The formatting looks right.   The code here looks wrong though.  😄    

I found 5 cases like this in the source. 
src\host\conareainfo.cpp(187):        SMALL_RECT OldRegion = _caInfo.rcViewCaWindow;;
src\host\consoleInformation.cpp(388):    ci.Attributes = GenerateLegacyAttributes(attr);;
src\interactivity\base\InteractivityFactory.cpp(349):    NTSTATUS status = ApiDetector::DetectNtUserWindow(&level);;
src\interactivity\win32\screenInfoUiaProvider.cpp(534):    HRESULT hr = S_OK;;
src\interactivity\win32\UiaTextRange.cpp(151):    UiaTextRange* range = nullptr;;

I can make an issue to clean these up.

Reply:
I can fix them up in this PR, no worries. I already had fixed some of this type of error as part of the initial PR but must've glossed over these.

Reply:
I'd rather you not actually -- this pull request should be as close to "whitespace changes only" as possible. Personal preference. What do you think?

Reply:
Sure, I can do it in a separate PR then. I don't have a strong opinion one way or the other.